apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/ItemService.java

```diff
@@ -144,6 +156,54 @@ public void syncItems(List<NamespaceIdentifier> comparedNamespaces, List<ItemDTO
     }
   }
 
+
+  public void revokeItem(NamespaceTextModel model) {
+
+    String appId = model.getAppId();
+    Env env = model.getEnv();
+    String clusterName = model.getClusterName();
+    String namespaceName = model.getNamespaceName();
+    long namespaceId = model.getNamespaceId();
```

Comment:
please load the namespace and get the namespaceId instead of just using what's passed in since the namespaceId might not match the namespaceName

```java
NamespaceDTO namespace = namespaceAPI.loadNamespace(appId, env, clusterName, namespaceName);
```

Reply:
这里我是参考这个接口写的..
https://github.com/ctripcorp/apollo/blob/6d8686f66ee42440675c3e9447e27a1d33bf01e5/apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/ItemController.java#L58

如果这样写会导致不匹配的话 是不是上面这个接口也需要修改？

Reply:
是的，modifyItemsByText这个也有潜在的安全风险

Reply:
已修改